README.md

```diff
@@ -1,9 +1,17 @@
-<h1 align='center'>Life's a Beach</h1>
+<h1 align='center'>Life's a Beach 🏖️</h1>
 
 <p align='center'><i>Don't get washed away in the sea of terminology</i></p>
```

Comment:
Uncapitalise Don't please

Reply:
Can you please do it - I am no longer at my computer :)